### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.4.7 Focus Visible

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#670]
Mary Jo:

- No comment for now, but will need to watch progress on issue #51. 

Chris L: 

- Mary Jo and Mitch both commented on this being something to validate for in mobile. 
- The question is back to the discussion for WCAG2Mobile on developing content for their note
- Mary Jo: This isn't about what the focus cursor looks like, but about whether or not it is visible when an interactive component receives keyboard focus. In WCAG, this was trying to solve the problem where the authored content was turning off the keyboard focus indicator or making it match the background so it wouldn't be visible to the user. IMO, the question to answer here is whether that type of capability exists in the mobile space.
-  Mitch: In Android apps, pressing TAB with an external keyboard, I fairly often see instances of focus not visible. It can occur when focus moves to an outer container. I don't know what's happening in the code; this is just with black box testing on a stock Google Pixel.

**Comment by maryjom** [comment]
I'm closing this issue for now since there is no content in WCAG2Mobile on this SC yet.
